From 0532c27dd88d7bebac935dd1815f9c2cadda23d8 Mon Sep 17 00:00:00 2001
From: WinKarbik <karbainovs@mail.ru>
Date: Sun, 12 May 2019 14:27:17 +0300
Subject: [PATCH 2/2] samsung: Move all to vendor

* Delete obsolete LOCAL_PRELINK_MODULE
* Use PRODUCT_VENDOR_MOVE_ENABLED := true for moving

Signed-off-by: WinKarbik <karbainovs@mail.ru>
Change-Id: Ie9ece40762d038b6a3b10d73453bf99e343c5344
---
 audio/Android.mk                    | 4 ++++
 consumerir/Android.mk               | 2 +-
 fingerprint/bauth/Android.mk        | 4 ++++
 liblights/Android.mk                | 2 +-
 modemloader/Android.mk              | 4 ++++
 power/Android.mk                    | 4 ++++
 ril/libsecril-client-sap/Android.mk | 5 ++++-
 ril/libsecril-client/Android.mk     | 5 ++++-
 wifiloader/Android.mk               | 4 ++++
 9 files changed, 30 insertions(+), 4 deletions(-)

diff --git a/audio/Android.mk b/audio/Android.mk
index 64ada48..9f58ae5 100644
--- a/audio/Android.mk
+++ b/audio/Android.mk
@@ -59,6 +59,10 @@ LOCAL_MODULE_RELATIVE_PATH := hw
 
 LOCAL_MODULE_TAGS := optional
 
+ifeq ($(PRODUCT_VENDOR_MOVE_ENABLED),true)
+LOCAL_VENDOR_MODULE := true
+endif
+
 include $(BUILD_SHARED_LIBRARY)
 
 endif
diff --git a/consumerir/Android.mk b/consumerir/Android.mk
index 2df62d7..7ee460d 100644
--- a/consumerir/Android.mk
+++ b/consumerir/Android.mk
@@ -32,6 +32,6 @@ endif
 LOCAL_MODULE := consumerir.$(IR_HAL_SUFFIX)
 LOCAL_MODULE_RELATIVE_PATH := hw
 LOCAL_MODULE_TAGS := optional
-LOCAL_PROPRIETARY_MODULE := true
+LOCAL_VENDOR_MODULE := true
 
 include $(BUILD_SHARED_LIBRARY)
diff --git a/fingerprint/bauth/Android.mk b/fingerprint/bauth/Android.mk
index e4041da..0ff9f24 100644
--- a/fingerprint/bauth/Android.mk
+++ b/fingerprint/bauth/Android.mk
@@ -12,4 +12,8 @@ LOCAL_MODULE_RELATIVE_PATH := hw
 LOCAL_MODULE := fingerprint.$(TARGET_BOARD_PLATFORM)
 LOCAL_MODULE_TAGS := optional
 
+ifeq ($(PRODUCT_VENDOR_MOVE_ENABLED),true)
+LOCAL_VENDOR_MODULE := true
+endif
+
 include $(BUILD_SHARED_LIBRARY)
diff --git a/liblights/Android.mk b/liblights/Android.mk
index a1629f1..8a3e16d 100644
--- a/liblights/Android.mk
+++ b/liblights/Android.mk
@@ -40,7 +40,7 @@ LOCAL_STATIC_LIBRARIES := liblights_helper
 LOCAL_MODULE := lights.$(TARGET_BOOTLOADER_BOARD_NAME)
 LOCAL_MODULE_RELATIVE_PATH := hw
 LOCAL_MODULE_TAGS := optional
-LOCAL_PROPRIETARY_MODULE := true
+LOCAL_VENDOR_MODULE := true
 
 include $(BUILD_SHARED_LIBRARY)
 
diff --git a/modemloader/Android.mk b/modemloader/Android.mk
index 1e18b32..0295eb0 100644
--- a/modemloader/Android.mk
+++ b/modemloader/Android.mk
@@ -23,4 +23,8 @@ LOCAL_SHARED_LIBRARIES := liblog libcutils
 LOCAL_MODULE := modemloader
 LOCAL_MODULE_TAGS := optional
 
+ifeq ($(PRODUCT_VENDOR_MOVE_ENABLED),true)
+LOCAL_VENDOR_MODULE := true
+endif
+
 include $(BUILD_EXECUTABLE)
diff --git a/power/Android.mk b/power/Android.mk
index 53f60b4..590a897 100644
--- a/power/Android.mk
+++ b/power/Android.mk
@@ -29,6 +29,10 @@ LOCAL_MODULE := power.$(TARGET_BOOTLOADER_BOARD_NAME)
 LOCAL_MODULE_RELATIVE_PATH := hw
 LOCAL_MODULE_TAGS := optional
 
+ifeq ($(PRODUCT_VENDOR_MOVE_ENABLED),true)
+LOCAL_VENDOR_MODULE := true
+endif
+
 ifneq ($(TARGET_TAP_TO_WAKE_NODE),)
     LOCAL_CFLAGS := -DTARGET_TAP_TO_WAKE_NODE=\"$(TARGET_TAP_TO_WAKE_NODE)\"
 endif
diff --git a/ril/libsecril-client-sap/Android.mk b/ril/libsecril-client-sap/Android.mk
index b436531..45dfd0f 100755
--- a/ril/libsecril-client-sap/Android.mk
+++ b/ril/libsecril-client-sap/Android.mk
@@ -17,6 +17,9 @@ LOCAL_SHARED_LIBRARIES := \
 LOCAL_CFLAGS := 
 
 LOCAL_MODULE:= libsecril-client-sap
-LOCAL_PRELINK_MODULE := false
+
+ifeq ($(PRODUCT_VENDOR_MOVE_ENABLED),true)
+LOCAL_VENDOR_MODULE := true
+endif
 
 include $(BUILD_SHARED_LIBRARY)
diff --git a/ril/libsecril-client/Android.mk b/ril/libsecril-client/Android.mk
index 2898bb2..7040d2c 100755
--- a/ril/libsecril-client/Android.mk
+++ b/ril/libsecril-client/Android.mk
@@ -24,6 +24,9 @@ LOCAL_CFLAGS += -DSAMSUNG_NEXT_GEN_MODEM
 endif
 
 LOCAL_MODULE:= libsecril-client
-LOCAL_PRELINK_MODULE := false
+
+ifeq ($(PRODUCT_VENDOR_MOVE_ENABLED),true)
+LOCAL_VENDOR_MODULE := true
+endif
 
 include $(BUILD_SHARED_LIBRARY)
diff --git a/wifiloader/Android.mk b/wifiloader/Android.mk
index 9d27fb7..206b0d3 100644
--- a/wifiloader/Android.mk
+++ b/wifiloader/Android.mk
@@ -27,6 +27,10 @@ endif
 LOCAL_MODULE := wifiloader
 LOCAL_MODULE_TAGS := optional
 
+ifeq ($(PRODUCT_VENDOR_MOVE_ENABLED),true)
+LOCAL_VENDOR_MODULE := true
+endif
+
 include $(BUILD_EXECUTABLE)
 
 endif
-- 
2.17.1

